cross/java-11-openjdk/patches/002.rh1648644-java_access_bridge_privileged_security.patch

```diff
@@ -0,0 +1,20 @@
+--- openjdk/src/java.base/share/conf/security/java.security
++++ openjdk/src/java.base/share/conf/security/java.security
```

Comment:
I'm sure what this patch is supposed to do, but I don't think this patch is doing anything here, since there is no gnome shell or APIs to call

Reply:
Removed